app/v2/messaging/create-message.js

```diff
@@ -0,0 +1,23 @@
+const { v4: uuidv4 } = require('uuid')
+
+const createMessage = (event) => {
+  const { type, source, subject = undefined, data = undefined } = event
+  return {
+    body: {
+      specversion: '1.0',
+      type,
+      source,
+      id: uuidv4(),
+      time: new Date().toISOString(),
```

Comment:
What does .toISOString() mean/do?

Reply:
It changes the `Date` JS object into a nice neat string.

ISO format is this -> `YYYY-MM-DDTHH:mm:ss.sssZ`

Then when the events are consumed and stored downstream they're all in a consistent format.

Reply:
Thanks :)